beater/sophoscentralbeat.go

```diff
@@ -56,7 +56,7 @@ var (
 const (
 	// ServiceName is the name of the service
 	ServiceName = "sophoscentralbeat"
-	//Environment variable name for fully qualified beat name
+	// FQBeatNamevariable name for fully qualified beat name
 	FQBeatName = "FullyQualifiedBeatName"

```

Comment:
Can you remove this const and just use the one below? We want there to be only a single source of truth for the FQBeatName.

Reply:
FQBeatName only in environment module, like siem repo's ./internal/pkg/environment